### PR TITLE
feat: Add Preimage option to C-Chain config

### PR DESCRIPTION
**Pull request by stevekunk**
As suggested by @aaronbuchwald here
https://github.com/ava-labs/coreth/issues/165

I just kinda blindly did what aaron suggested. I've been busy. Not sure what the best way to test it or what documentation needs to be updated as a result of this.

### CODE REVIEW

**Comment by stevekunk** [comment]
I've never used golang heavily and it was years ago before "go modules" what was a thing. If anyone has any suggestions on how to update just coreth to my fork, I can update my example in to show if this works as expected or not.

File: plugin/evm/config.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ const (
 	defaultEthApiEnabled               = true
 	defaultNetApiEnabled               = true
 	defaultWeb3ApiEnabled              = true
+	defaultPreimagesEnabled            = true
```
Comment (aaronbuchwald) [inline plugin/evm/config.go:18]: Can you remove this line? We should allow this to default to false.

**Review by aaronbuchwald — CHANGES_REQUESTED** [review]
Can you change the behavior so that the default for `Preimages` is false? Then I think this will be good to merge.

File: plugin/evm/config.go (hunk 1 above)
Comment (stevekunk) [reply plugin/evm/config.go]: Done in 3bf563221

**Pull request merged**